conf/iridanext.config

```diff
@@ -10,6 +10,9 @@ iridanext {
                 "**/arborator/cluster_summary.tsv",
                 "**/arborator/metadata.included.tsv",
                 "**/arborator/metadata.excluded.tsv",
+                "**/arborator/*_metadata.tsv",
```

Comment:
Could you include in here `*_clusters.tsv`, `*_loci.summary.tsv`, `*_tree.nwk`, `*_matrix.tsv`, and `*_outliers.tsv`?

Reply:
Sure, thing. I didn't realize we wanted those outputs.

Updated in ca13d77fb0a8f3fa3cfb08e213d8f0ee53f6c44d